src/index.html

```diff
@@ -3,11 +3,370 @@
+            <p><span class="menu__opening--bold">11:00 — 19:00</span> неділя</p>
+          </div>
+
+          <div class="menu__article--right">
+            <span class="menu__opening--bold">Вихідні:</span> понеділок, вівторок
+            <br>
+            Kaca припиняє роботу за 1 годину
+            до закриття музею
+          </div>
+        </article>
+      </div>
+    </div>
+
+    <main class="main">
+      <section class="events" id="events">
+        <h2 class="title title__secondary events__title">
+          Актуальні події
+        </h2>
+
+        <img src="./images/events/events_image1.jpg"
+          alt="events-image"
+          class="picture events__picture"
+          data-aos="fade-up"
+        >
+
+        <div class="events__content">
+          <p class="events__date">ВИСТАВКА <span class="events__date--highlight">26.08-29.11.2019</span></p>
+
+          <h3 class="title title__third title--rectangle">
+            Йду i повертаюсь
+          </h3>
+
+          <p class="events__info">
+            Національний Художній Музей України презентує унікальну частину колекції Градобанку - українське мистецтво
+            другої половини 1980-1995 років.
+          </p>
+        </div>
+
+        <img
+          src="./images/events/events_image2.jpg"
+          alt="events-image"
+          class="picture events__picture"
+          data-aos="fade-up"
+        >
+
+        <div class="events__content">
+          <p class="events__date">ВИСТАВКА <span class="events__date--highlight">26.08-29.11.2019</span></p>
+
+          <h3 class="title title__third title--rectangle">
+            I спогади i мрії
+          </h3>
+
+          <p class="events__info">
+            Національний художній музей України до 100 річчя від дня народження видатної української художниці Тетяни
+            Яблонської відкриває ретроспективну ювілейну виставку «І спогади i мрії».
+          </p>
+        </div>
+      </section>
+
+      <section class="separator">
+        <div class="separator__image"></div>
+
+        <div class="separator__frame">
+          <p class="separator__date">ЛЕКЦІЯ <span class="separator__date--highlight">26/08/2019-11:00</span></p>
+
+          <h3 class="title title--separator">
+            Від класицизму до романтизму
+          </h3>
+        </div>
+      </section>
+
+      <section class="gallery" id="gallery">
+        <h2 class="title title__secondary">
+          Галерея
+        </h2>
+
+        <div class="gallery container-gallery">
+          <img src="./images/gallery/gallery_image1.jpg"
+            alt="gallery-image"
+            class="gallery-desktop
+            gallery-desktop__image
+            gallery-desktop__image--1"
+          >
+
+          <img src="./images/gallery/gallery_image2.jpg"
+            alt="gallery-image"
+            class="gallery-desktop
+            gallery-desktop__image
+            gallery-desktop__image--2"
+          >
```

Comment:
Let's fix code style everywhere
```suggestion
          <img
            src="./images/gallery/gallery_image2.jpg"
            alt="gallery-image"
            class="gallery-desktop
              gallery-desktop__image
              gallery-desktop__image--2"
          >
```